pkg/proxy/userspace/proxier.go

```diff
@@ -260,8 +263,10 @@ func CleanupLeftovers(ipt iptables.Interface) (encounteredError bool) {
 
 // Sync is called to immediately synchronize the proxier state to iptables
 func (proxier *Proxier) Sync() {
-	if err := iptablesInit(proxier.iptables); err != nil {
-		glog.Errorf("Failed to ensure iptables: %v", err)
+	if runtime.GOOS != "windows" {
```

Comment:
Yeah, I'd agree that there are too many:

`if runtime.GOOS != windows`

for my liking, I'd prefer to see both iptables and netsh implement some sort of `init()` interface and just call:"

`if iptables != nil { iptables.init() }`

or something like that...